static/flow/flow-util.js

```diff
@@ -98,4 +98,61 @@ Util.getLocalTime = function(d) {
     return Util.getLocalDateTime(d)[1];
 };
 
+//
+// Util to convert date/time string to human readable form
+// input ex: 20180522_164244
+// output ex: December 22, 2018 10:55 PM
+//
+Util.convertDateTimeStringToHumanReadable = function(dateTimeStr) {
```

Comment:
It seems unnecessary for us to write our own time format/parser instead of taking advantage of something like JS `Date`s...but I can see that least moving this function is an improvement

Reply:
Yeah, probably better to pull in something that handles this.  I thought the same thing at some point, but didn't get around to doing it.